src/Giraffe/Giraffe.fsproj

```diff
@@ -54,16 +54,16 @@
     <IncludeAssets>build</IncludeAssets>
     <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="G-Research.FSharp.Analyzers" Version="0.10.0">
+    <PackageReference Include="G-Research.FSharp.Analyzers" Version="0.11.0">
```

Comment:
Updating analyzers is best handled in a separate PR, to be honest. It can make the current PR noisy, and I generally find that smaller PRs focusing on one thing are easier to review.